src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/PrefsFramework.md

```diff
@@ -0,0 +1,775 @@
+</figure>
+
+* This Editor is a full-page editor, so all of what you see is Preferences Editor
+(the outer, blue dashed line).
+* The content inside the rectangle (outlined with a green dashed line) is called a Panel –
+a container representing one (or more) preferences. This particular Editor has only one Panel,
+but a realistic Editor will likely have several. This tutorial will teach you what you need to
+know to add more Panels to this Editor.
+* Inside the Panel is an Adjuster (outlined by the inner-most, orange dashed line) – the controls
+for adjusting a particular preference. This Panel has only one Adjuster in it, but you might want
+to create a Panel that has multiple Adjusters, say, in the case of very closely-related preferences.
+This tutorial will teach you about creating Panels with multiple Adjusters.
+
+Let’s take a close look at the code.
+
+#### Primary Schema ####
+The [Primary Schema](../PrimarySchemaForPreferencesFramework.md) is a document that defines the preferences for the Editor.
+The Primary Schema for our example Editor is defined in the
+`schemas/primary.js` file using the JSON format
+(you can learn about JSON at http://json.org/):
+
+```javascript
+/**
+ * Primary Schema
+ * This schema defines the preference(s) edited by this preference editor:
+ * their names, types, default values, etc.
+ */
+fluid.defaults("minEditor.primarySchema", {
+
+    // the base grade for the schema;
+    // using this grade tells the framework that this is a primary schema
+    gradeNames: ["fluid.prefs.schemas"],
+
+    schema: {
+        // the actual specification of the preference
+        "minEditor.autoPilot": {
+            "type": "boolean",
+            "default": false
+        }
+    }
+});
+```
+
+In this code snippet, the Primary Schema is created using a call to the
+Infusion Framework function `fluid.defaults().`
+
+<aside class="infusion-docs-callout">
+`fluid.defaults()` is one of the core functions in Infusion: It is used to create [components](../UnderstandingInfusionComponents.md)
+(the building blocks of any Infusion application) and register them with the Framework.
+</aside>
+
+`fluid.defaults()` accepts two arguments:
+1. a string name, and
+2. a JavaScript object containing [options for configuring](../ComponentConfigurationOptions.md) the component.
+
+In the code snippet above, the first argument – the name – is `“minEditor.primarySchema”`.
+The second argument – the options – is an object containing (in this case) two properties:
+`gradeNames` and `schema`:
+
+<dl>
+<dt>`gradeNames`</dt>
+<dd>
+Any call to `fluid.defaults()` must include the `gradeNames` property in the options argument.
+This property defines the base _[grade](../ComponentGrades.md)_ for the component.
+<aside class="infusion-docs-callout">
+A **grade** is _very loosely_ analogous to a class, in that using a grade in the definition of a
+component infers the properties of that grade to the component. It’s actually a bit more complex
```

Comment:
Misuse of "infers" here. Better to say "the properties of the component are derived from the elements in the grade document"